test/unit/source/MVT.test.js

```diff
@@ -61,5 +62,43 @@ describe('sources/MVT', () => {
                 metadata.codec('wadus').sourceToInternal(123);
             }).toThrowError(/MVT decoding error. Metadata property is of type \'category\' but the MVT tile contained a feature property of type 'number': \'123\'/);
         });
+        it('Category property should work with undefined value', () => {
+            const metadata = new MVTMetadata({
+                properties: {
+                    wadus: {
+                        type: 'category'
+                    }
+                },
+                idProperty: 'id'
+            });
+            metadata.setCodecs();
+            expect(metadata.codec('wadus').sourceToInternal(undefined)).toEqual(FP32_DESIGNATED_NULL_VALUE);
+        });
+        it('Category property should work with null value', () => {
+            const metadata = new MVTMetadata({
+                properties: {
+                    wadus: {
+                        type: 'category'
+                    }
+                },
+                idProperty: 'id'
+            });
+            metadata.setCodecs();
+            expect(metadata.codec('wadus').sourceToInternal(null)).toEqual(FP32_DESIGNATED_NULL_VALUE);
```

Comment:
We have 'undefined' and 'null'... how about considering **empty string** (with trim)?

Reply:
An empty string will resolve to the same internal ID and the `typeof` won't be undefined. I think it's safe.